visualdl/logic/im.cc

```diff
@@ -30,10 +39,23 @@ void SimpleWriteSyncGuard<T>::Start() {
 
 template <typename T>
 void SimpleWriteSyncGuard<T>::End() {
-  CHECK(data_);
-  if (data_->parent()->meta.ToSync()) {
-    Sync();
-  }
+    CHECK(data_);
```

Comment:
The c++ code should follow google c style, indent 2 spaces not 4 spaces.

`clang-format` can format c++ code automatically.